salt/states/netacl.py

```diff
@@ -428,7 +427,11 @@ def term(name,
             - filter_options:
                 - not-interface-specific
             - term_name: {{ term_name }}
-            - {{ my_term_cfg }}
+            - {{ my_term_cfg | json }}
+
+    When passing retrieved pillar data into the state file, it is strongly
+    recommended to use the json serializer explicitly, instead of relying
```

Comment:
So just a single note near the top of the page referencing the updated examples?  I'm open to it, but won't most people jump straight to the function they're trying to utilize?